### PR TITLE
FedLearner Framework and Core Dependency RA-TLS Configuration

### PR DESCRIPTION
**Pull request by henshy**
1. FedLearner Framework and Core Dependency RA-TLS Configuration
To ensure code consistency and implement hash signature-based remote attestation for the FedLearner framework and its core dependencies, an environment variable has been introduced in Gramine that prevents the generation of .pyc files.

2. Gramine Template Configuration for FedLearner
The Gramine template configuration has been updated to include the code locations for the FedLearner framework and some essential dependencies. This addition facilitates bidirectional remote attestation between parties.

3. Meituan HDFS File Path Management Optimization
The code responsible for handling file paths in Meituan's Hadoop Distributed File System (HDFS) has been migrated from the main entry point to be processed by the master node. This change aims to prevent file read and write conflicts that could arise from multiple workers operating simultaneously.

### CODE REVIEW

File: fedlearner-sgx-dev.dockerfile [hunk 1]
```diff
@@ -60,8 +62,8 @@ RUN apt-get install -y libcurl4-openssl-dev libprotobuf-c-dev python3-protobuf p
 RUN apt-get install -y libgmp-dev libmpfr-dev libmpc-dev libisl-dev nasm protobuf-compiler
 
 RUN ln -s /usr/bin/python3 /usr/bin/python \
-    && pip3 install --upgrade pip \
-    && pip3 install toml meson pyelftools
+    && pip3 install --no-compile --upgrade pip -i https://mirrors.aliyun.com/pypi/simple/ \
```
Comment (zeuson0) [inline fedlearner-sgx-dev.dockerfile:65]: Is it necessary to use mirror from aliyun?

File: setup.py [hunk 2]
```diff
@@ -11,8 +11,7 @@
 
 def get_version():
     base = "1.5"
-    day = time.strftime('%Y%m%d', time.localtime())
-    return '%s-dev%s'%(base, day)
+    return '%s-dev' % base
```
Comment (zeuson0) [inline setup.py:14]: What the change here for?

File: fedlearner-sgx-dev.dockerfile (hunk 1 above)
Comment (henshy) [reply fedlearner-sgx-dev.dockerfile:65]: If we do not include Aliyun, the compilation and testing on our side are too slow. Therefore, we have added it to anything related to pip, which also ensures the consistency of dependency sources.

File: setup.py (hunk 2 above)
Comment (henshy) [reply setup.py:14]: If the time is included, the resulting dist-info package path for FedLearner will contain the date, causing the path to change with each compilation, which makes it impossible to add to the Gramine configuration.

**Pull request merged**